src/graphorm/index.ts

```diff
@@ -74,6 +77,9 @@ const obj = new GraphORM({
       notificationEmail: {
         transform: nullIfNotSameUser,
       },
+      createdAt: {
+        transform: transformDate,
+      },
```

Comment:
The parent entity of this field is the `type User`. I have a feeling that after adding this, the issue with having `createdAt` on our FE will be resolved.

Reply:
Isnt it the generic typeorm date issue we have?